icons/intellij/intellij-original.svg

```diff
@@ -1 +1 @@
a_intellij-idea_svg__a" x1="5.174" x2="40.014" y1="39.889" y2="38.123" gradientUnits="userSpaceOnUse"><stop offset=".091" stop-color="#FC801D"></stop><stop offset=".231" stop-color="#B07F61"></stop><stop offset=".409" stop-color="#577DB3"></stop><stop offset=".533" stop-color="#1E7CE6"></stop><stop offset=".593" stop-color="#087CFA"></stop></linearGradient><linearGradient id="intellij-idea_intellij-idea_svg__b" x1="61.991" x2="50.158" y1="36.915" y2="1.557" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#FE2857"></stop><stop offset=".078" stop-color="#CB3979"></stop><stop offset=".16" stop-color="#9E4997"></stop><stop offset=".247" stop-color="#7557B2"></stop><stop offset=".339" stop-color="#5362C8"></stop><stop offset=".436" stop-color="#386CDA"></stop><stop offset=".541" stop-color="#2373E8"></stop><stop offset=".658" stop-color="#1478F2"></stop><stop offset=".794" stop-color="#0B7BF8"></stop><stop offset="1" stop-color="#087CFA"></stop></linearGradient><linearGradient id="intellij-idea_intellij-idea_svg__c" x1="10.066" x2="53.876" y1="16.495" y2="88.96" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#FE2857"></stop><stop offset=".08" stop-color="#FE295F"></stop><stop offset=".206" stop-color="#FF2D76"></stop><stop offset=".303" stop-color="#FF318C"></stop><stop offset=".385" stop-color="#EA3896"></stop><stop offset=".553" stop-color="#B248AE"></stop><stop offset=".792" stop-color="#5A63D6"></stop><stop offset="1" stop-color="#087CFA"></stop></linearGradient></defs><path fill="url(&quot;#intellij-idea_intellij-idea_svg__a&quot;)" d="M11.2 49.4668L0.699951 41.3001L9 26L18.5 33.5L11.2 49.4668Z"></path><path fill="#087CFA" d="M69.9999 18.6666L68.8333 59.2666L41.7666 70L27.0666 60.4333L41.7666 37.5L69.9999 18.6666Z"></path><path fill="url(&quot;#intellij-idea_intellij-idea_svg__b&quot;)" d="M70 18.6666L55.5 33L37 15L48.0666 1.16663L70 18.6666Z"></path><path fill="url(&quot;#intellij-idea_intellij-idea_svg__c&quot;)" d="M27.0667 60.4333L5.6 68.3667L10.0333 52.5L15.8667 33.1333L0 27.7667L10.0333 0L33.1333 2.8L54.5 31L55.5 33L27.0667 60.4333Z"></path><path fill="#000" d="M56 14H14V56H56V14Z"></path><g fill="#FFFFFF"><path d="M27.1366 22.1433V19.25H19.2733V22.1433H21.4666V32.1067H19.2733V34.9767H27.1366V32.1067H24.92V22.1433H27.1366Z" fill="#FFFFFF"></path><path d="M34.6967 35.21C33.46 35.21 32.4334 34.9767 31.6167 34.51C30.7767 34.0433 30.1 33.4833 29.5634 32.8533L31.7334 30.4267C32.1767 30.9167 32.6434 31.3133 33.0867 31.5933C33.5534 31.8733 34.0434 32.0133 34.6034 32.0133C35.2567 32.0133 35.77 31.8033 36.1434 31.3833C36.5167 30.9633 36.7034 30.31 36.7034 29.4V19.2733H40.25V29.5633C40.25 30.4967 40.1334 31.3133 39.8767 32.0133C39.62 32.7133 39.2467 33.2967 38.78 33.7633C38.29 34.2533 37.7067 34.6033 37.0067 34.86C36.3067 35.0933 35.5367 35.21 34.6967 35.21Z" fill="#FFFFFF"></path><path d="M34.4166 48.6499H18.6666V51.3332H34.4166V48.6499Z" fill="#FFFFFF"></path></g></symbol></defs><use xlink:href="#intellij-idea"></use></svg>
```

Comment:
[Viewbox should be 0 0 128 128](https://github.com/devicons/devicon/wiki/SVG-Standards)

Reply:
The same applies to all the icons you submitted^